packaging/debs/Debian/debian/control

```diff
@@ -7,11 +7,12 @@ Uploaders: Alvaro Videla <alvaro@rabbitmq.com>,
  Jean-Sébastien Pédron <jean-sebastien@rabbitmq.com>,
  Giuseppe Privitera <giuseppe@rabbitmq.com>
 Build-Depends: debhelper (>= 9),
+ dh-systemd (>= 1.5),
```

Comment:
Actually 14.04 was working just fine, as dh-systemd in trusty is 1.14
But I'm not sure about debian oldstable.